diff --git a/src/G5.Logic/BotGameState.cs b/src/G5.Logic/BotGameState.cs
index 8c4a664..6d4ce03 100644
--- a/src/G5.Logic/BotGameState.cs
+++ b/src/G5.Logic/BotGameState.cs
@@ -82,7 +82,7 @@ namespace G5.Logic
             _randomlySampleActions = randomlySampleActions;
 
             string assemblyFolder = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
-            _preFlopCharts = new PreFlopCharts(assemblyFolder + "\\PreFlopCharts\\200bb\\");
+            _preFlopCharts = new PreFlopCharts(Path.Combine(assemblyFolder, "PreFlopCharts", "200bb"));
 
             _tableType = tableType;
             _players = new List<Player>();
